src/components/Article.jsx

```diff
@@ -1 +1,30 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = (
+  { title, date, text },
+) => {
```

Comment:
better destructure props inside of components, this will make future expanding easier
```suggestion
export const Article = (props) => {
```